apps/snitch_core/test/data/model/variant_test.exs

```diff
@@ -0,0 +1,34 @@
+defmodule Snitch.Data.Model.VariantTest do
+  use ExUnit.Case, async: true
+  use Snitch.DataCase
+
+  import Snitch.Factory
+
+  alias Snitch.Data.Model.Variant
+
+  test "get_category/1" do
```

Comment:
@oyeb this test should become changes as per my above change suggestion in the name of the function.